brave/ui/app/components/app/connect-wallet/connect-wallet.component.js

```diff
@@ -0,0 +1,78 @@
+import React,{ PureComponent } from "react";
+import styles from './assets/styles'
+
+module.exports = class ConnectWallet extends PureComponent {
+
+  get text () {
```

Comment:
I wouldn't use a `get` accessor for this since it creates react elements whenever it is accessed, and that isn't something that'd want to be done more often than neccessary. I would use a function called `renderText`

Reply:
done